sharding-core/sharding-core-common/src/test/java/org/apache/shardingsphere/core/rule/ShardingDataSourceNamesTest.java

```diff
@@ -90,4 +91,15 @@ public void assertGetDefaultDataSourceNameWithoutMasterSlaveDataSourceName() {
         String actual = new ShardingDataSourceNames(shardingRuleConfig, Arrays.asList("master_ds", "slave_ds")).getRawMasterDataSourceName("default_ds");
         assertThat(actual, is("default_ds"));
     }
+
+    @Test
```

Comment:
```suggestion
    @Test(expected = IllegalArgumentException.class)
```

will free you from catching exceptions manually and determining their types